src/events/discord/message.ts

```diff
@@ -17,13 +17,17 @@ discord.on("ready", async () => {
 discord.on("message", async (message: Message) => {
   /* Intent: MESSAGE_READ
    * Purpose: Kindly remind user not to mention whitep4nth3r.
+   *
+   * todo: clean this up. 3 nested ifs is very smelly.
    */
   if (message.content.includes(`<@!${config.discord.whitep4nth3rId}>`)) {
-    if (message.author.id !== config.discord.p4nth3rb0tId) {
-      message.channel.send(
-        `Attention <@!${message.author.id}>! Please refrain from mentioning ` +
-        `whitep4nth3r directly if it is not extremely urgent. Thank you.`,
-      );
+    if(!config.discord.excludedChannels.indexOf(message.channel.id)) {
```

Comment:
From what I understand, in the case that the message is sent in the first channel in the list of excluded channels, the index will be `0`, and `!0` will resolve to `true`.

Maybe this could work?
```suggestion
    if(!config.discord.excludedChannels.includes(message.channel.id)) {
```